src/Domain/CampoKeys.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Fau\DegreeProgram\Common\Domain;
+
+use InvalidArgumentException;
+
+final class CampoKeys
+{
+    public const SCHEMA = [
+        'type' => 'object',
+        'properties' => [
+            DegreeProgram::DEGREE => [
+                'type' => 'string',
+            ],
+            DegreeProgram::AREA_OF_STUDY => [
+                'type' => 'string',
+            ],
+            DegreeProgram::LOCATION => [
+                'type' => 'string',
+            ],
+        ],
+    ];
+
+    public const SCHEMA_REQUIRED = [
+        'type' => 'object',
+        'properties' => [
+            DegreeProgram::DEGREE => [
+                'type' => 'string',
+            ],
+            DegreeProgram::AREA_OF_STUDY => [
+                'type' => 'string',
+            ],
+            DegreeProgram::LOCATION => [
+                'type' => 'string',
+            ],
+        ],
+    ];
```

Comment:
Is it intentional that the required schema is identical to the draft one? Should we require those properties, or should we not require Campo key parts for terms?

Reply:
Since entering campokeys for terms is optional, I used the same schema for required as well, otherwise it can prevent saving the degree program if one of the terms doesn't have a campo key.